sofascoreAcademy/sofascoreAcademy/Views/CustomNavigationTitleView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CustomNavigationTitleView.swift
+//  sofascoreAcademy
+//
+//  Created by Tina Jureško on 03.04.2025..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class CustomNavigationTitleView: BaseView {
+    private let leagueLogoImageView = AsyncImageView()
+    private let titleLabel = UILabel()
+    
+    override func addViews() {
+        super.addViews()
+        addSubview(leagueLogoImageView)
+        addSubview(titleLabel)
+    }
+
+    override func styleViews() {
+        leagueLogoImageView.contentMode = .scaleAspectFit
+        titleLabel.font = .regular14
```

Comment:
Postavi style za ovo, i onda zaključi da li imaš nešto u setupConstrints() što ti ne treba?